cmd/minikube/cmd/delete.go

```
@@ -46,10 +48,27 @@ associated files.`,
 
 // runDelete handles the executes the flow of "minikube delete"
 func runDelete(cmd *cobra.Command, args []string) {
-	if len(args) > 0 {
-		exit.Usage("usage: minikube delete")
+	profileFlag, _ := cmd.Flags().GetString("profile")
```

let's not ignore errors